src/Symfony/Component/Messenger/Transport/Schedule/ScheduleTransport.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Schedule;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Exception\LogicException;
+use Symfony\Component\Messenger\Exception\TransportException;
+use Symfony\Component\Messenger\Stamp\ScheduledStamp;
+use Symfony\Component\Messenger\Transport\TransportInterface;
+
+class ScheduleTransport implements TransportInterface
+{
+    public function __construct(private readonly \Traversable $schedule)
```

Comment:
Let's create an interface instead:

```
interface ScheduleInterface
{
    public function getSchedule(): iterable;
}
```

Reply:
I did it :rofl: I can return it, if you wish.

The idea, why I removed it is: the message it's any object of any class, and the schedule it's an iterable of that objects. So, it's possible to do any iterable and pass it to transport to provide messages. And it will work.